### PR TITLE
CI

### PR DESCRIPTION
**Pull request by tokatoka**

### CODE REVIEW

**Comment by domenukk** [comment]
```rust
  --- stderr
  thread 'main' panicked at libafl_concolic/symcc_runtime/build.rs:151:10:
  Unable to generate bindings: NotExist("symcc/runtime/rust_backend/RustRuntime.h")
```
wtf, this one is new

**Comment by domenukk** [comment]
Maybe the PATH is messed up now, or something?

**Comment by domenukk** [comment]
Could also be a sideeffect of exporting `DOCS_RS=1`?

**Comment by tokatoka** [comment]
> Could also be a sideeffect of exporting DOCS_RS=1?

yes

File: scripts/parallellize_cargo_check.py [hunk 1]
```diff
@@ -34,4 +32,8 @@
     instance_idx * 2 * task_per_core : (instance_idx + 1) * 2 * task_per_core
 ]:
     print("Running ", task)
-    cargo_check = subprocess.check_output(task, shell=True, text=True)
+    print(os.environ)
+    if "libafl_frida" in task:
+        cargo_check = subprocess.check_output(task, shell=True, text=True, env=dict(os.environ, DOCS_RS="1"))
```
Comment (domenukk) [inline scripts/parallellize_cargo_check.py:37]: Might be a good idea to comment why we do this for future reference
(`# DOCS_RS is needed for libafl_frida to build without auto-download feature`)

**Pull request merged**